templates/default/event/eventmodify.html

```diff
@@ -115,14 +115,11 @@ <H1>{t a=$event.ticketid}Add new event to request #$a{/t}</H1>
 					{box_row icon="img/users.gif" label="Users:" field_class="multiselect"}
 						<SELECT name="event[userlist][]" class="lms-ui-multiselect" multiple data-default-value="{trans("- select user -")}" data-shorten-to-default-value="false">
 							{foreach $userlist as $user}
-							{if !ConfigHelper::checkConfig('phpui.timetable_hide_disabled_users') ||
-							(ConfigHelper::checkConfig('phpui.timetable_hide_disabled_users') && $user.access) ||
-							in_array($user.id, $filter.owner)}
-							<OPTION {if !$user.access} disabled{/if} value="{$user.id}"
-							{if is_array($event.userlist) && in_array($user.id, $event.userlist)}}selected{/if}>
-								{$user.rname|truncate:40:"...":true}({$user.login})
-							</OPTION>
-							{/if}
+								{if !ConfigHelper::checkConfig('phpui.timetable_hide_disabled_users') || (ConfigHelper::checkConfig('phpui.timetable_hide_disabled_users') && $user.access) || in_array($user.id, $filter.owner)}
```

Comment:
Akurat warunki można rozbić na kilka wierszy - następne wiersze z dodatkowym wcięciem. Nie możesz po prostu spojrzeć na przykłady. Szkoda czasu to wałkować po wielokroć.

Reply:
Zrobione wraz z rebase - w webeditorze akurat to nie wygląda za dobrze.

Reply:
A w PHPStorm przestawiłeś sobie typ pliku dla html z HTML na Smarty?
Pisałem o tym w którymś komentarzu, że może być istotne - na pewno wtedy rozpoznaje składnię Smarty i do tego możliwe, że inaczej kod formatuje.

Reply:
Zobacz teraz.